modules/qna/src/translations/en.json

```diff
@@ -88,6 +90,7 @@
     "uploadFile": "Upload File",
     "uploadStatus": "Upload status",
     "uploadSuccessful": "Upload successful",
+    "uploading": "Uploading",
```

Comment:
Can you put single words translations in `ui-shared` instead ? We will just end up translating them multiple times otherwise.